core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -23,14 +23,21 @@
  * {@link RuntimeException} if the key is not found in both configurations..
  */
 public final class UnderFileSystemConfiguration {
+  private final boolean mReadOnly;
+  private final boolean mShared;
```

Comment:
@calvinjia @apc999  I'm wrapping `MountInfo` in `UnderFileSystemConfiguration`. Let me know if you feel we you feel we should rid of the class `UnderFileSystemConfiguration` completely.